source/default/_patterns/00-protons/demo/demo-block.twig

```diff
@@ -0,0 +1,25 @@
+<div class="container">
```

Comment:
I can be swayed here, but wanted to run it by you:

The only "rule" we've laid down for the difference between protons and atoms is that

>Protons are not directly usable in Twig via embed/extend/include

This will be the first proton that breaks that rule. Ideas/counter points?

Reply:
Sure. I think for me the view on protons are that they're mainly design configuration or work that supports Pattern Lab. I wouldn't include this pattern as a part of my design implementation, but it's useful in the terms of Particle to help demonstrate a use. 

Secondly, placing it in Protons means it is _never_ swept into an application as a useable pattern. I'd never want someone to use this within Drupal or Grav.

Lastly, I think it's a bad usage to include a pattern that's created later in the chain. I think the second place this makes sense is templates, but I find it problematic for an atom to include/embed/extend a template. We've tried that before and it really complicates deleting unused patterns.

Reply:
**THE MOTION PASSES**